### PR TITLE
Add card design and license with attribution in readme

### PR DESCRIPTION
**Pull request by dbertella**

### CODE REVIEW

File: packages/client/src/components/Card.tsx [hunk 1]
```diff
@@ -22,45 +21,30 @@ function name([value, suit]: CardType) {
...
-const suitList = Object.values(SUIT);
-function bgPosition([value, suit]: CardType) {
-  return `${(value - 1) * 11}% ${suitList.indexOf(suit) * 33}%`;
+function imgUrl([value, suit]: CardType) {
+  return require(`../images/${suit.toLowerCase()}/${padStart(`${value}`, 2, '0')}.jpg`).default;
 }
 
-const face = {
+const baseCard = {
   height: '13vw',
```
Comment (stathismor) [inline packages/client/src/components/Card.tsx:29]: Do we resize the assets here? Like, scaling down/up?

File: packages/client/src/components/Card.tsx [hunk 2]
```diff
@@ -22,45 +21,30 @@ function name([value, suit]: CardType) {
   return `${VALUES[value]} of ${SUIT[startCase(suit)]}`;
 }
 
-const suitList = Object.values(SUIT);
-function bgPosition([value, suit]: CardType) {
-  return `${(value - 1) * 11}% ${suitList.indexOf(suit) * 33}%`;
+function imgUrl([value, suit]: CardType) {
+  return require(`../images/${suit.toLowerCase()}/${padStart(`${value}`, 2, '0')}.jpg`).default;
```
Comment (stathismor) [inline packages/client/src/components/Card.tsx:25]: Could you use `import` here? Manually construct the name as you do, import it, and return it.

File: packages/client/src/components/Card.tsx [hunk 3]
```diff
@@ -22,45 +21,30 @@ function name([value, suit]: CardType) {
...
-
+function cardStyle(card: CardType, faceDown?: boolean) {
+  return {
+    ...baseCard,
+    backgroundImage: `url(${faceDown ? back : imgUrl(card)})`,
+  };
+}
 interface CardProps {
-  className?: string;
   faceDown?: boolean;
```
Comment (stathismor) [inline packages/client/src/components/Card.tsx:44]: Should this just always exist?

**Review by stathismor — APPROVED** [review]

File: packages/client/src/components/Card.tsx (hunk 1 above)
Comment (dbertella) [reply packages/client/src/components/Card.tsx:29]: we are not resizing, just fixing a height of the div seen on screen, the real image size won't change

File: packages/client/src/components/Card.tsx (hunk 2 above)
Comment (dbertella) [reply packages/client/src/components/Card.tsx:25]: I don't think you can import dinamically, this is a weird case, we will replace with links once we have a cdn but for now it should be ok with require

File: packages/client/src/components/Card.tsx (hunk 3 above)
Comment (dbertella) [reply packages/client/src/components/Card.tsx:44]: no it's fine to have it undefined, so you add it only when you need it. Will see in the future if we need something else

Comment (stathismor) [reply packages/client/src/components/Card.tsx:44]: Yeah, I just prefer more explicit definitions and defaulting the values (as you have done) rather that maybes.

**Pull request merged**